Sources/SwiftParser/Attributes.swift

```diff
@@ -573,6 +571,15 @@ extension Parser {
           trailingComma: comma,
           arena: self.arena
         )))
+      case .none:
+        let valueLabel = self.consumeAnyToken()
```

Comment:
Let's try to parse a trailing comma here just in case the user wrote

```
@_specialize(e, exported: true)
```

or something.

Reply:
Thanks for the feedback Robert! I've pushed another commit.